redaxo/src/core/lib/packages/addons/addon.php

```diff
@@ -232,6 +232,7 @@ public function getSystemPlugins()
             $systemPlugins = (array) $this->getProperty('system_plugins', []);
         }
         $plugins = [];
+        /* @var string[] */
```

Comment:
das ist so kein gültiger phpdoc, jedenfalls aus sicht des cs-fixers. deswegen hat er es auch in einen normalen comment umgewandelt.

Üblich ist über foreach-schleifen die hintere var zu notieren

```php
/** @var string $plugin */
foreach ($systemPlugins as $plugin) {
```

Reply:
Hmm ich hatte im Hinterkopf dass der phpcs-fixer die vorgeschlagene notation auch nicht mag.. mal probieren https://github.com/redaxo/redaxo/pull/3442